Observation/Observation-CellularityKayBurbridge-Example.json

```diff
@@ -0,0 +1,39 @@
+{
+  "resourceType": "Observation",
+  "id": "Observation-CellularityKayBurbridge-Example",
+  "status": "final",
+  "code": {
+    "coding": [
+      {
+        "system": "https://loinc.org/",
+        "code": "55437-8",
+        "display": "Neoplastic cells/100 leukocytes"
```

Comment:
I would also switch this out for text "Cellularity" as I don't think this directly relates to neoplastic cells